app/src/main/java/com/example/airbnb/presentation/when/viewholder/CalenderViewHolder.kt

```diff
@@ -0,0 +1,24 @@
+package com.example.airbnb.presentation.`when`.viewholder
+
+import android.content.Context
+import androidx.core.content.ContextCompat
+import androidx.recyclerview.widget.RecyclerView
+import com.example.airbnb.R
+import com.example.airbnb.databinding.ItemCalenderDateBinding
+
+class CalenderViewHolder(
+    private val context: Context,
+    private val binding: ItemCalenderDateBinding,
+    private val onCalenderDateClick: (String, Int) -> Unit = { _, _ -> }
+) : RecyclerView.ViewHolder(binding.root) {
+    fun onBind(position: Int, data: String) {
+        if (position % 7 == 0) {
+            binding.tvCalenderDate.setTextColor(
+                ContextCompat.getColor(
+                    context, R.color.main1
+                )
+            )
```

Comment:
저 전부터 궁금했는데, 어떤 곳에서는 그냥 Color.black 이런 식으로 values에 있는 컬러값을 바로 불러와서 할당이 가능한데, 어떤 곳에서는 이런식으로 context까지 쓰면서 getColor 함수를 쓰던데, 어떤 경우에 이런 코드를 쓰는지 알 수 있을까요!

Reply:
```suggestion
            binding.tvCalenderDate.setTextColor(Color.parseColor("#DD0F62"))
```
전자는 요런 코드인데, 이 코드는 하드 코딩된 색상 값을 사용하여 TextView의 텍스트 색상을 설정하는 것입니다!(Black..등 기본 색상은 이미 Color 클래스에 지정되어 있기 때문에 Color.BLACK 이런식으로 사용할 수 있는 것! 그러나 우리가 쓰는 컬러는 따로 parseColor로 지정해주어야 되겠쥬? )

제가 쓴 코드 즉 후자의 코드는 리소스 파일에서 색상을 가져와서 설정하는 방법입니다!
요 방법을 사용한 이유는 리소스 파일을 활용해서 설정하는 것이 더 일관성 있다고 생각되어서!! (매번 하드 코딩된 색상 값을 가져와서 쓴다면 나중에 컬러 변경이 있을 경우 다 바꿔주어야 하는 불상사가 생기겠죠?)